Mathlib/AlgebraicGeometry/AffineScheme.lean

```diff
@@ -578,6 +578,15 @@ theorem isLocalization_stalk (x : U) :
   subst this
   exact hU.isLocalization_stalk' y hx
 
+lemma stalkMap_injective_of (f : X ⟶ Y) {U : Opens Y} (hU : IsAffineOpen U) (x : X)
```

Comment:
I don't think the trailing `_of` is necessary? Same for `stalkMap_injective_of_isAffine_of`.